src/sctools/bam.py

```diff
@@ -260,8 +265,8 @@ def _cleanup(files_to_counts, files_to_names, rm_all=False) -> None:
         warnings.warn('Number of requested subfiles (%d) exceeds 500; this may cause OS errors by '
                       'exceeding fid limits' % n_subfiles)
     if n_subfiles > 1000:
-        raise ValueError('Number of requested subfiles (%d) exceeds 1000; this will usually cause '
-                         'OS errors, think about increasing max_mb_per_split.' % n_subfiles)
+        raise ValueError(f'Number of requested subfiles ({ n_subfiles}) exceeds 1000; this will usually cause '
```

Comment:
Why the leading space before `n_subfiles`?

Reply:
Fixed.

Reply:
Also, extracted constants.